### PR TITLE
AIP: Events and Sponsorships Grant 2024

### PR DESCRIPTION
**Pull request by CheyenneAtapour**
https://governance.aave.com/t/arfc-aave-events-sponsorship-proposal-2024/18276

### CODE REVIEW

File: src/20240718_AaveV3Ethereum_EventsGrant2024/AaveV3Ethereum_EventsGrant2024_20240718.t.sol [hunk 1]
```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {AaveV3Ethereum, AaveV3EthereumAssets} from 'aave-address-book/AaveV3Ethereum.sol';
+
+import 'forge-std/Test.sol';
+import {ProtocolV3TestBase, ReserveConfig} from 'aave-helpers/ProtocolV3TestBase.sol';
```
Comment (sendra) [inline src/20240718_AaveV3Ethereum_EventsGrant2024/AaveV3Ethereum_EventsGrant2024_20240718.t.sol:7]: Reserve config is not being used

File: src/20240718_AaveV3Ethereum_EventsGrant2024/EventsGrant2024.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+---
+title: "Events Grant 2024"
+author: "Aave Labs"
+discussions: "https://governance.aave.com/t/arfc-aave-events-sponsorship-proposal-2024/18276"
+snapshot: "https://snapshot.org/#/aave.eth/proposal/0x5d4e3fba58f76f516afd0855a687027270b74163911116f14a4f5c01c34a9bd9"
+---
+
+## Summary
+
+This AIP proposes the Aave DAO budget allocation on events at key ecosystem initiatives at EthCC and Devcon to help to reinforce Aave Protocol’s unique and positive culture, share technical knowledge, and attract new community members. These initiatives are aimed at helping to continue the expansion of the DeFi ecosystem by showcasing Aave Protocol’s core values. Thank you for reviewing this proposal and for your earlier questions and positive vote in the snapshot.
```
Comment (sendra) [inline src/20240718_AaveV3Ethereum_EventsGrant2024/EventsGrant2024.md:10]:
```suggestion
This AIP proposes the Aave DAO budget allocation on events at key ecosystem initiatives at EthCC and Devcon to help to reinforce Aave Protocol’s unique and positive culture, share technical knowledge, and attract new community members. These initiatives are aimed at helping to continue the expansion of the DeFi ecosystem by showcasing Aave Protocol’s core values.
```

File: src/20240718_AaveV3Ethereum_EventsGrant2024/EventsGrant2024.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+After covering the aforementioned main sponsorships, any remaining budget will be directed towards side events related to the security of DeFi and smart contract technology. Any excess funds from this grant will be rolled over to be used for events during Q1 2025. A recap on last year’s events was posted to governance last December, here.
+Aave Labs shall cover all expenses for its team members, and this proposal does not request any funds for these expenses, or include any compensation for Aave Labs’ work.
+
```
Comment (sendra) [inline src/20240718_AaveV3Ethereum_EventsGrant2024/EventsGrant2024.md:34]: Missing here, to which address this is sent, and also that the amount is transferred from the collector

File: lib/aave-helpers
Comment (ianflexa) [inline lib/aave-helpers:1]: should be sync with main

**Review by sendra — APPROVED** [review]

**Review by ianflexa — APPROVED** [review]

**Pull request merged**